autogpts/autogpt/poetry.lock

```diff
@@ -333,13 +344,13 @@ uvicorn = "^0.23.2"
 webdriver-manager = "^4.0.1"
 
 [package.extras]
-benchmark = ["agbenchmark @ git+https://github.com/Significant-Gravitas/AutoGPT.git#subdirectory=benchmark"]
+benchmark = ["agbenchmark @ file:///home/mk/.cache/pypoetry/virtualenvs/agpt-xgev2_OR-py3.11/src/AutoGPT/benchmark"]
```

Comment:
This happens to me for some reason as well, please revert this to git address and be careful when updating poetry packages.

Reply:
Oh sure thanks